Description: Do not set skb truesize since using one linearskb
 From Linux bb6a777:
 .
 We are seeing this warning: at net/core/skbuff.c:4174
 and before commit a44878d10063 ("IB/ipoib: Use one linear skb in RX flow")
 skb truesize was not being set when ipoib was using just one skb.
 Removing this line avoids the warning when running tcp tests like iperf.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Author: Carol L Soto <clsoto@linux.vnet.ibm.com>
Origin: https://github.com/torvalds/linux/commit/bb6a777369449d15a4a890306d2f925cae720e1c#diff-354653dff7ec9ddd6ac73f85f2956316
Forwarded: not-needed
Last-Update: <2017-12-14>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ib_ipoib/ipoib_ib.c
+++ b/ib_ipoib/ipoib_ib.c
@@ -245,8 +245,6 @@
 	skb_reset_mac_header(skb);
 	skb_pull(skb, IPOIB_ENCAP_LEN);
 
-	skb->truesize = SKB_TRUESIZE(skb->len);
-
 	++dev->stats.rx_packets;
 	dev->stats.rx_bytes += skb->len;
 
